ui/Screen/Project/Commits.svelte

```diff
@@ -0,0 +1,103 @@
+<script>
+  import { format } from "timeago.js";
+  import { link } from "svelte-spa-router";
+
+  import * as path from "../../lib/path.js";
+  import {
+    commits as store,
+    fetchCommits,
+    formatTime
+  } from "../../src/source.ts";
+
+  import { Remote } from "../../DesignSystem/Component";
+  import { Text } from "../../DesignSystem/Primitive";
+  import CommitTeaser from "../../DesignSystem/Component/SourceBrowser/CommitTeaser.svelte";
+
+  export let params = null;
+
+  const projectId = params.id;
+  const branch = params.branch;
+
+  fetchCommits({ projectId, branch });
+</script>
+
+<style>
+  .commits-page {
+    padding-top: 32px;
+    margin-bottom: 64px;
+    margin-left: 96px;
+    margin-right: 96px;
+    min-width: 720px;
+    position: relative;
+  }
+  .commit-group header {
+    padding-bottom: 0.75rem;
+    margin-left: 1.25rem;
+    padding-left: 1rem;
+    color: var(--color-foreground-level-6);
+  }
+  .commit-group ul {
+    border: 1px solid var(--color-foreground-level-3);
+    border-radius: 4px;
+  }
+  .commit-group:after {
+    content: "";
+    height: 2rem;
+    left: 1.25rem;
+    position: relative;
+    display: block;
+  }
+  .commit-group:after,
+  .commit-group header {
+    border-left: 2px solid var(--color-foreground-level-3);
+  }
+  .commit-group:first-child header {
+    border-left: none;
```

Comment:
I understand the idea. But I feel or I would remove the border in general or leave it at the top for now. This seems broken.

Reply:
Like this without the line.
<img width="1058" alt="Screenshot 2020-05-05 at 11 52 13" src="https://user-images.githubusercontent.com/2326909/81054319-debfc500-8ec6-11ea-97ca-d26abd7f294c.png">